### PR TITLE
Site power controller uses HELICS

### PR DESCRIPTION
**Pull request by haitamlaarabi**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/3621)

### CODE REVIEW

File: src/main/python/gemini/site_power_controller_federate.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    logging.info("beam_to_pydss_federate finalized")
+
+    h.helicsFederateFree(cfed)
+    h.helicsCloseLibrary()
+
+
+###############################################################################
+
+
+data = pd.read_csv("../../../../production/sfbay/parking/sfbay_taz_unlimited_charging_point.csv")
```
Comment (nikolayilyin) [inline src/main/python/gemini/site_power_controller_federate.py]: Would be great to put it as a parameter to a main function, or, at least, to check if the file exist and throw a meaningful exception if not.

**Review by nikolayilyin — APPROVED** [review]
lgtm

**Review by nikolayilyin — APPROVED** [review]
Would be great to move files like src/main/R/freight/speed_validation_NPMRDS.ipynb from src/ folder somewhere to jupyter/
The rest looks good to me

**Comment by nikolayilyin** [comment]
test!

**Comment by haitamlaarabi** [comment]
Test now!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by alex-vv** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by alex-vv** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by haitamlaarabi** [comment]
Test now!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
A copy of this branch before merging develop was created (#3664)

**Pull request merged**